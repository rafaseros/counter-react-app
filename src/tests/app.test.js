import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import App from '../app';

describe('Prueba de App.jsx', () => {
  // test('should be show h1 and subtitle', () => {
  //   // const saludo = 'Hola soy Rafael';
  //   // const { getByText } = render(<App saludo={saludo} />);
  //   // expect(getByText(saludo)).toBeInTheDocument();
  // });

  test('should be show App', () => {
    const saludo = 'hola soy rafael';
    const wrapper = shallow(<App saludo={saludo} />);

    expect(wrapper).toMatchSnapshot();
  });
});
