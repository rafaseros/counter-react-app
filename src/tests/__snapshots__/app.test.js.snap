// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de App.jsx should be show App 1`] = `
<Fragment>
  <h1>
    hola soy rafael
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
